dotnet/src/Skills/Skills.UnitTests/Skills.UnitTests.csproj

```diff
@@ -42,12 +42,8 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\SemanticKernel\SemanticKernel.csproj" />
-    <ProjectReference Include="..\Skills.Core\Skills.Core.csproj" />
-    <ProjectReference Include="..\Skills.Document\Skills.Document.csproj" />
     <ProjectReference Include="..\Skills.Grpc\Skills.Grpc.csproj" />
```

Comment:
Should this file be renamed/removed?

Reply:
These two projects:

```
..\Skills.Grpc\Skills.Grpc.csproj
..\Skills.OpenAPI\Skills.OpenAPI.csproj
```

are going to be treated differently. Current thinking is that these will become:

```
..\Functions.Grpc\Functions.Grpc.csproj
..\Functions.OpenAPI\Functions.OpenAPI.csproj
```